psi4/driver/qcdb/gradparse.py

```diff
@@ -0,0 +1,94 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2018 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+
+import numpy as np
+
+from .util import filter_comments
+
+# TODO: Include gradient loading feature, analagously to hessian loading.
+
+def to_string(grad, handle, dtype='file11', mol=None, energy=None):
```

Comment:
Can we change this to `gradeint_to_string`, again to be very explicit?

Reply:
That would break mirroring of `hessparse.py`, which also has a `to_string` method. I can rename that one to `hessian_to_string`, while I'm at it.

Reply:
can we just always use it namespaced `gradparse.to_string()` unless in-class? there's a few of these -- `hessparse.to_string()`, `molparse.to_string()`, etc.

Reply:
Good point. The namespace would make the renaming redundant, and the namespacing is standard anyway.